### PR TITLE
DEC-662: Pages

### PR DESCRIPTION
**Pull request by jgondron**
Why: Users need to view pages for a collection
How: 
-Added page cards to collection home page
-Added page renderers with item viewers for any embedded items
-Updated beehive to use site_path instead of just showcases for determining flow/order of pages and showcases
-Made some minor tweaks to title bars on showcases to be more consistent across pages and showcases.

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

File: src/components/Collection/CollectionIntro.jsx [hunk 1]
```diff
@@ -20,7 +18,7 @@ var CollectionIntro = React.createClass({
   render: function() {
     if(this.props.collection) {
       return (
-        <PagesShow content={this.props.collection.short_description } />
+        <div className="essay-content" style={this.style()} dangerouslySetInnerHTML={{__html:this.props.collection.short_description}} />
```
Comment (rfox2) [inline src/components/Collection/CollectionIntro.jsx]: I don't know - this line looks too dangerous.

File: src/layout/PreviewLink.jsx [hunk 2]
```diff
@@ -32,18 +42,55 @@ var PreviewLink = React.createClass({
       color: "#f5f5f5",
       opacity: this.state.hover ? "1.0" : "0.3",
       maxWidth: "200px",
+    };
+    switch(media){
```
Comment (rfox2) [inline src/layout/PreviewLink.jsx]: Is this doing anything?

File: src/mixins/CollectionUrlMixin.jsx [hunk 3]
```diff
@@ -15,19 +15,11 @@ var CollectionUrlMixin = {
...
-    return this.collectionObjectUrl('showcases', showcase);
-  },
-
-  firstShowcaseUrl: function(showcase) {
+  startSitePathUrl: function() {
     var url;
-    if(this.props.collection.showcases){
-      if(this.props.collection.showcases.length > 0) {
-        url = this.showcaseUrl(this.props.collection.showcases[0]);
+    if(this.props.collection.site_path){
```
Comment (rfox2) [inline src/mixins/CollectionUrlMixin.jsx]: Do we need the double conditional here?

**Comment by rfox2** [comment]
Just a couple of questions, but otherwise :+1:

File: src/components/Collection/CollectionIntro.jsx (hunk 1 above)
Comment (danthewolfe) [inline src/components/Collection/CollectionIntro.jsx]: Sometimes you need to live dangerously.

File: src/mixins/CollectionUrlMixin.jsx (hunk 3 above)
Comment (jgondron) [inline src/mixins/CollectionUrlMixin.jsx]: We do need both conditionals, but I'm assuming you mean do we need two if blocks. I can move it into one.

File: src/layout/PreviewLink.jsx (hunk 2 above)
Comment (jgondron) [inline src/layout/PreviewLink.jsx]: Yes, it jumps to the "wide" case when media == "wide", and jumps to the "narrow" case when media == "narrow". Aside from that, no. I'll remove :trollface:

**Pull request merged**